cmd/edenPod.go

```diff
@@ -253,28 +270,31 @@ func newPodLogsCmd(cfg *openevec.EdenSetupArgs) *cobra.Command {
 }
 
 func newPodModifyCmd(cfg *openevec.EdenSetupArgs) *cobra.Command {
+	var podNetworks, portPublish, acl, vlans []string
+	var startDelay uint32
+
 	var podModifyCmd = &cobra.Command{
 		Use:   "modify <app>",
 		Short: "Modify pod",
 		Args:  cobra.ExactArgs(1),
 		Run: func(cmd *cobra.Command, args []string) {
 			appName := args[0]
-			if err := openevec.PodModify(appName, cfg); err != nil {
+			if err := openevec.PodModify(appName, podNetworks, portPublish, acl, vlans, startDelay, cfg); err != nil {
 				log.Fatalf("EVE pod start failed: %s", err)
 			}
 		},
 	}
 
-	podModifyCmd.Flags().StringSliceVarP(&cfg.Runtime.PortPublish, "publish", "p", nil, "Ports to publish in format EXTERNAL_PORT:INTERNAL_PORT")
-	podModifyCmd.Flags().BoolVar(&cfg.Runtime.ACLOnlyHost, "only-host", false, "Allow access only to host and external networks")
```

Comment:
We don't use this argument in PodModify function. Is it correct behaviour @giggsoff

Reply:
It is correct, the logic of pod modify is quite limited now.